baseplate/clients/thrift.py

```diff
@@ -145,7 +146,7 @@ def _call_thrift_method(self: Any, *args: Any, **kwargs: Any) -> Any:
         trace_name = f"{self.namespace}.{name}"
         last_error = None
 
-        for _ in self.retry_policy:
+        for value in self.retry_policy:
```

Comment:
since the value emitted by `self.retry_policy` is either time_remaining or none, let's just call it `time_remaining` instead of `value` for better readability.